### PR TITLE
Update rust-rocksdb and change branch

### PR DESCRIPTION
**Pull request by yiwu-arbug**
## What have you changed? (mandatory)
Update rocksdb to with the fix:
```
b4f33387f 2019-06-20 yiwu@pingcap.com     Fix ingested file and direcotry not being sync (#96)
```
Also point change rust-rocksdb to point to tikv-3.0 branch.

## What are the type of the changes? (mandatory)

Bug fix & Engineering

## How has this PR been tested? (mandatory)

CI

## Does this PR affect documentation (docs) or release note? (mandatory)

no

## Does this PR affect tidb-ansible update? (mandatory)

no

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
 ## [Unreleased]
++ Engine
+    - Synchronize all files and directories for received snapshots [#4853](https://github.com/tikv/tikv/pull/4853)
```
Comment (yiwu-arbug) [inline CHANGELOG.md]: This is copy paste from 2.1.12 changelog.

Comment (Connor1996) [reply CHANGELOG.md]: please change PR link

**Pull request merged**